src/dmd/mars.d

```diff
@@ -245,21 +246,37 @@ private int tryMain(size_t argc, const(char)** argv)
         return EXIT_SUCCESS;
     }
 
-    if (global.params.mcpuUsage)
+    /*
+    Prints a supplied usage text to the console and
+    returns the exit code for the help usage page.
+
+    Returns:
+        `EXIT_SUCCESS` if no errors occurred, `EXIT_FAILURE` otherwise
+    */
+    static byte printHelpUsage(string help)
```

Comment:
Should be `int` because `EXIT_SUCCESS`, etc., are defined as `int`.

Reply:
FWIW they aren't: https://github.com/dlang/druntime/blob/a59c3832fc00ab127018d9a570ba03520437524f/src/core/stdc/stdlib.d#L80 (they are `enum`s)